src/transformers/models/transfo_xl/modeling_transfo_xl.py

```diff
@@ -1137,6 +1137,19 @@ def _resize_cutoffs(self, new_num_tokens, new_emb_size, new_embedding_shapes, la
         self.crit.cutoff_ends = [0] + new_cutoffs
         self.crit.n_token = new_num_tokens
 
+    @staticmethod
+    def _reorder_cache(mems: List[torch.Tensor], beam_idx: torch.Tensor) -> List[torch.Tensor]:
+        """
+        This function is used to re-order the :obj:`past_key_values` or :obj:`mems` cache if
+        :meth:`~transformers.PretrainedModel.beam_search` or :meth:`~transformers.PretrainedModel.beam_sample` is
+        called. This is required to match :obj:`past_key_values` or :obj:`mems` with the correct beam_idx at every
+        generation step.
+
+        For custom re-ordering of :obj:`past_key_values` or :obj:`mems`, the function should be implemented in
```

Comment:
remove those lines and `past_key_values` above

Reply:
I cleaned it as well.